chronosphere/tfschema/log_allocation_config.go

```diff
@@ -0,0 +1,49 @@
+package tfschema
+
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+const maxAllocations = 10
+
+var LogAllocationConfig = map[string]*schema.Schema{
+	"default_dataset": {
+		Type: schema.TypeList,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"allocation": LogAllocationSchema,
+			},
+			SchemaVersion: 1,
+		},
+		Required: true,
+		MaxItems: 1,
+	},
+	"dataset_allocation": {
```

Comment:
Made this singular to align with resource pools updated [schema](https://github.com/chronosphereio/terraform-provider-chronosphere/blob/2c97566d1fcb0c66c2840c4e9bb4890aa8efc4fe/chronosphere/tfschema/resource_pools_config.go#L42) but happy to make plural if thats preferred.

Reply:
i think that seems reasonable

Reply:
does it clash with the naming schema of anything else?

Reply:
We intentionally went with a naming scheme similar to resource pools. But don't think it should be a technical issue right?